sbt-scalajs-bundler/src/main/scala/scalajsbundler/util/ScalaJSOutputAnalyzer.scala

```diff
@@ -0,0 +1,57 @@
+package scalajsbundler.util
+
+import org.scalajs.core.ir.Trees.JSNativeLoadSpec
+import org.scalajs.core.tools.io.VirtualScalaJSIRFile
+import org.scalajs.core.tools.linker._
+import org.scalajs.core.tools.linker.standard._
+import org.scalajs.core.tools.linker.backend.{BasicLinkerBackend, LinkerBackend}
+import org.scalajs.sbtplugin.Loggers
+import org.scalajs.sbtplugin.ScalaJSPlugin.AutoImport.ModuleKind
+import sbt.Logger
+
+object ScalaJSOutputAnalyzer {
```

Comment:
This commit introduces `ScalaJSOutputAnalyzer` but does not remove `ReloadWorkflow.findImportedModules`. It should do so.